### PR TITLE
Fix "run iOS on device" scenario 

### PR DESCRIPTION
**Pull request by vladimir-kotikov**

### CODE REVIEW

**Comment by vladimir-kotikov** [comment]
@iTOYS PTAL

File: src/common/ios/iOSPlatform.ts [hunk 1]
```diff
@@ -183,8 +183,11 @@ export class IOSPlatform extends GeneralMobilePlatform {
     }
 
     private generateSuccessPatterns(): Q.Promise<string[]> {
-        return this.getBundleId().then(bundleId =>
-            IOSPlatform.RUN_IOS_SUCCESS_PATTERNS.concat([`Launching ${bundleId}\n${bundleId}: `]));
+        return this.targetType === "device" ?
```
Comment (iTOYS) [inline src/common/ios/iOSPlatform.ts]: Maybe check with `IOSPlatform.deviceString`is better.

Comment (iTOYS) [reply src/common/ios/iOSPlatform.ts]: Otherwise LGTM

Comment (vladimir-kotikov) [reply src/common/ios/iOSPlatform.ts]: Yup, fixed

**Pull request merged**